src/compiler/transformers/create-event.ts

```diff
@@ -4,10 +4,18 @@ import ts from 'typescript';
 import type * as d from '../../declarations';
 import { addCoreRuntimeApi, CREATE_EVENT, RUNTIME_APIS } from './core-runtime-apis';
 
-export const addCreateEvents = (moduleFile: d.Module, cmp: d.ComponentCompilerMeta) => {
+/**
+ * For a Stencil component, generate the code to create custom emitted events, based on `@Event()` decorators
+ * @param moduleFile the component class representation that code is being generated for
```

Comment:
possibly should be

```suggestion
 * @param moduleFile the 'home module' of the class for which code is being generated
```

Reply:
👍 I felt like my brain didn't want to make a coherent sentence when I wrote this. Much better

Reply:
some days it be like that haha